assets/js/services/global-vars.js

```diff
@@ -7,3 +7,70 @@ export const IIIF_SIZES = {
   IIIF_SQUARE: "/square/500,500/0/default.jpg",
   IIIF_FULL: "/full/full/0/default.jpg"
 };
+export const PRESERVATION_LEVELS = [
+  { label: "Level 1", id: 1 },
+  { label: "Level 2", id: 2 },
+  { label: "Level 3", id: 3 }
+];
+export const RIGHTS_STATEMENTS = [
+  {
+    id: " http://rightsstatements.org/vocab/InC/1.0/",
+    label: "In Copyright",
+    active: true
```

Comment:
@katdivyareddy10 Sorry I didn't catch this before, but can we remove the `active: true` field on all objects here?   Not sure what it does, but guessing it got copied and pasted over from the Donut object.